pyspecProcScripts/correlation_alignment.py

```diff
@@ -170,15 +178,21 @@ def correl_align(
                     ax=ax_list[1],
                 )
                 ax_list[1].set_title("correlation function (t), \nafter apod")
+        # we want Δnu centered at 0 and don't care about the previous window
+        # selection
         correl.ft_new_startpoint(direct, "freq")
         correl.ft_new_startpoint(direct, "time")
+        # Part of convolution is multiplying in t domain and then zero-filling
+        # before re-FT
         correl.ft(direct, shift=True, pad=2**14)
-        for k, v in signal_pathway.items():
-            correl.ft(["Delta%s" % k.capitalize()])
+        # {{{ Isolate the correlation function for the signal pathway
+        for sig_pathname, sig_pathval in signal_pathway.items():
+            correl.ft(["Delta%s" % sig_pathname.capitalize()])
             correl = (
-                correl["Delta" + k.capitalize(), v]
-                + correl["Delta" + k.capitalize(), 0]
+                correl["Delta" + sig_pathname.capitalize(), sig_pathval]
+                + correl["Delta" + sig_pathname.capitalize(), 0]
```

Comment:
line 192 to 193 here are attempting to apply a mask and sum along the Δp_l dimension.

You should comment that the ft on line 190 transforms from Δφ to Δp_l